modules/atomic-widgets/atomic-dynamic-tags/module.php

```diff
@@ -0,0 +1,146 @@
+<?php
+
+namespace Elementor\Modules\AtomicWidgets\AtomicDynamicTags;
+
+use Elementor\Modules\AtomicWidgets\Controls\Section;
+use Elementor\Modules\AtomicWidgets\Controls\Types\Select_Control;
+use Elementor\Modules\AtomicWidgets\Controls\Types\Text_Control;
+use Elementor\Modules\AtomicWidgets\Schema\Atomic_Prop;
+use Elementor\Modules\AtomicWidgets\Schema\Constraints\Enum;
+
+class Module {
+	public function register_hooks() {
+		add_filter( 'elementor/editor/localize_settings', fn( $settings ) => $this::add_atomic_dynamic_tags_settings( $settings ) );
+	}
+
+	private function add_atomic_dynamic_tags_settings( $settings ) {
+		$settings['atomicDynamicTags'] = self::transform_dynamic_tags_to_atomic( $settings['dynamicTags']['tags'] );
+
+		return $settings;
+	}
+
+	public static function transform_dynamic_tags_to_atomic( $dynamic_tags ) {
```

Comment:
Maybe?
I would avoid using the word transform so people won't get confused with the data transformers.

```suggestion
	public function convert_to_atomic_dynamic_tags( $dynamic_tags ) {
```